.github/workflows/editorconfig.yml

```diff
@@ -2,6 +2,7 @@ name: actions
 
 on:
   pull_request:
+    types: [ opened, synchronize ]
```

Comment:
It should already more or less of what you are proposing:

```
Note: By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.
```

Reply:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
somehow I missed that 😅